resources/core/charts/helm-broker/charts/etcd/templates/_helpers.tpl

```diff
@@ -1,32 +1,10 @@
 {{/* vim: set filetype=mustache: */}}
-{{/*
-Expand the name of the chart.
-*/}}
-{{- define "etcd.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
-
-{{/*
+ {{/*
```

Comment:
remove that file, same as we did in service-catalog/etcd, hardoce a name as we did in https://github.com/kyma-project/kyma/blob/6606b10f49e9ca5d3125629da31a463b2465b8c5/resources/core/charts/service-catalog/charts/etcd/templates/etcd-cluster-cr.yaml

Reply:
but in this PR https://github.com/kyma-project/kyma/pull/92 I also changed the way how the etcd is installed for SC